helm/happa/values.yaml

```diff
@@ -39,6 +39,7 @@ happa:
   host: ""
   letsencrypt: ""
   featureFlags:
+    mapiAuth: false
     mapiClusters: false
```

Comment:
Does one feature make sense without the other?

Reply:
Yes

Reply:
mapiClusters also cannot be used on AWS, while mapiAuth can be used on all providers